code/procs/scanprocs.dm

```diff
@@ -198,7 +198,7 @@
 
 
 	var/datum/statusEffect/simpledot/radiation/R = M.hasStatus("radiation")
-	if (R)
+	if (R && R.stage)
```

Comment:
R?.stage moment

Reply:
`if(null && null.property)` won't runtime because the first check evaluates false and the second check is thus never run

Reply:
oh wait, you were saying I could replace this whole thing with `R?.stage`. I am dum.